webpack.config.js

```diff
@@ -17,7 +17,7 @@
  */
 
 // Depends
-var _ = require('lodash');
+var merge = require('webpack-merge');
```

Comment:
Hi! Sorry for the long silence and thanks for comment!

I just saw him in big projects and decided to try too. But your choice is better! Fixed